charts/logging-operator-logging/values.yaml

```diff
@@ -63,6 +63,9 @@ tls:
   fluentbitSecretName: ""
   fluentdSecretName: ""
 
+# Whether to skip invalid Flow and ClusterFlow resources
+skipInvalidResources: false
```

Comment:
Not sure about the position of this flag in file. The CRD (https://github.com/banzaicloud/logging-operator/blob/master/config/crd/bases/logging.banzaicloud.io_loggings.yaml) defines it right before `watchNamespaces`, so I figured I would do the same.